build.cake

```diff
@@ -99,26 +94,32 @@ Task("Clean")
 Task("Restore")
     .Does(() =>
 {
-    PaketRestore();
-
-    var msBuildSettings = new MSBuildSettings { ToolPath = msBuildPath, ArgumentCustomization = args => args.Append("/m") };
-    MSBuild(solution, msBuildSettings
-            .SetConfiguration(configuration)
-            .SetVerbosity(Verbosity.Minimal)
-            .WithTarget("restore")
-            );
+    var msBuildSettings = new MSBuildSettings {
+        Verbosity = Verbosity.Minimal,
+        ToolVersion = MSBuildToolVersion.VS2017,
```

Comment:
Why do we have to specify this explicitly?

Reply:
If you mean the tool version, this was added so we don't need the additional tool dependency on "VSwhere". If you mean the MSBuildSettings, they were already there and I find this way more easy to read as the fluent calls which where there. It separates the how and what nicely.

Reply:
I was talking about the toolversion.
You can safely remove the explicit toolversion as the previous usage of vswhere was mistakenly added by @punker76 as far as i remember.

Regarding the other settings, why do we need to state the PlatformTarget here?

Reply:
I'm not sure if it's the default, but we specify the configuration and the platform target together.
Maybe it's just because I like to be "verbose" and show what I want...

You decide what you want, without ToolVersion and PlatformTarget?
Not sure it works...

Reply:
Being explicit here would require a change here when we want to use VS2019 ;-)